docker_templates/templates/docker_images/ros2_entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# setup ros2 environment
+source "/opt/ros/$ROS2_DISTRO/setup.bash"
```

Comment:
do you think we could share the ROS entrypoint? or maybe it will add complexity just for the sake of deduplicating 3 lines...